test/unit/sessions/base-session-handler.test.ts

```diff
@@ -670,6 +671,28 @@ describe('addMediatoSession', () => {
     expect(mockSession.pc.addTrack).toHaveBeenCalled();
   });
 
+  it('should not add track if it\'s already there', async () => {
+    const stream = new MockStream(true);
+    const track = new MockTrack('audio');
+    stream._tracks = [track];
+    jest.spyOn(mediaUtils, 'checkHasTransceiverFunctionality').mockReturnValue(true);
+
+    const mockSession: any = {
+      pc: {
+        addTrack: jest.fn(),
+        addStream: jest.fn(),
+        getSenders: jest.fn().mockReturnValue([ { track }])
+      }
+    };
+
+    const logSpy = jest.spyOn(handler as any, 'log');
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9907734)
```suggestion
    const logSpy = jest.spyOn(handler as unknown, 'log');
```